Makefile

```diff
@@ -99,16 +99,17 @@ imports: $(OPENSHIFT_GOIMPORTS)
 
 COUNT ?= 5
 E2E_PARALLELISM ?= 1
+TEST_ARGS ?= ""
 
 .PHONY: test-e2e
 test-e2e: WHAT ?= ./test/e2e...
 test-e2e: install
-	go test -race -count $(COUNT) -p $(E2E_PARALLELISM) -parallel $(E2E_PARALLELISM) $(WHAT)
+	go test -race -count $(COUNT) -p $(E2E_PARALLELISM) -parallel $(E2E_PARALLELISM) $(TEST_ARGS) $(WHAT)
```

Comment:
Might want to remove `COUNT` and `E2E_PARALLELISM`? Maybe? IDK. /shrug

Reply:
I'm ok to leave them for now. We can revisit later if needed.